funnel/models/event.py

```diff
@@ -66,14 +66,14 @@ class Event(ScopedNameTitleMixin, db.Model):
     """
     __tablename__ = 'event'
 
-    proposal_space_id = db.Column(None, db.ForeignKey('proposal_space.id'), nullable=False)
-    proposal_space = db.relationship(ProposalSpace,
+    project_id = db.Column(None, db.ForeignKey('project.id'), nullable=False)
+    project = db.relationship(Project,
```

Comment:
It's possible to define `relationship("Project")` without the need of importing Project in all models file. Should we make that a part of our dev guideline?

Reply:
It is possible, but SQLAlchemy uses `eval` to process it. It is safer to use the direct object reference.